index.js

```diff
@@ -4,35 +4,49 @@
  * Copyright (c) 2015 David da Silva
  * Licensed under the MIT license.
  */
+ /* global define: false */
+(function (global, factory) {
+  if (typeof exports === 'object' && exports && typeof exports.nodeName !== 'string') {
+    factory() // CommonJS
+  } else if (typeof define === 'function' && define.amd) {
+    define([], factory) // AMD
+  } else {
+    global.i18n = factory() // script, wsh, asp
+  }
+}(this, function mustacheI18nFactory () {
 
-'use strict'
+  'use strict'
 
-var dict
-var _lookup
+  var dict
+  var _lookup
 
-var i18n = module.exports = function (mustache) {
-  var Context = mustache.Context
-  _lookup = Context.prototype.lookup
-  Context.prototype.lookup = function i18nLookup (name) {
-    return (name === 'i18n' ? i18n.lambda : _lookup.apply(this, arguments))
+  var i18n = function (mustache) {
+    var Context = mustache.Context
+    _lookup = Context.prototype.lookup
+    Context.prototype.lookup = function i18nLookup (name) {
+      return (name === 'i18n' ? i18n.lambda : _lookup.apply(this, arguments))
+    }
+    return i18n
   }
-  return i18n
-}
 
-i18n.lambda = function i18nLambda (text, render) {
-  if (!dict) throw new Error('i18n dict was not set up, do `i18n.use(dict)`.')
-  text = dict[text] || text
-  return render(text)
-}
+  i18n.lambda = function i18nLambda (text, render) {
+    if (!dict) throw new Error('i18n dict was not set up, do `i18n.use(dict)`.')
+    text = dict[text] || text
+    return render(text)
+  }
 
-i18n.use = function setUpDict (newDict) {
-  if (!_lookup) {
-    throw new Error('i18n was not properly set up, pass in `Mustache`:\n' +
-      '```js\n' +
-      'var Mustache = require(\'mustache\')\n' +
-      'var i18n = require(\'mustache-i18n\')(Mustache)\n' +
-      '```')
+  i18n.use = function setUpDict (newDict) {
+    if (!_lookup) {
+      throw new Error('i18n was not properly set up, pass in `Mustache`:\n' +
+        '```js\n' +
+        'var Mustache = require(\'mustache\')\n' +
+        'var i18n = require(\'mustache-i18n\')(Mustache)\n' +
+        '```')
+    }
+    dict = newDict
   }
-  dict = newDict
-}
 
+  if (module.exports) module.exports = i18n
```

Comment:
Evaluating this condition in the browser will blow up, with something along the lines of:
```
Uncaught ReferenceError: module is not defined(…)
```

Make it just `return i18n`, and in the CommonJS initialisation branch do:
```js
module.exports = factory()
```